WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift

```diff
@@ -0,0 +1,94 @@
+import UIKit
+
+final class WeatherViewController: UIViewController {
+
+    lazy private var cityTextField: UITextField = UITextField()
```

Comment:
Although we haven't had a convention for the order of access control yet, we can refer the raywenderlich's access control [convention](https://github.com/raywenderlich/swift-style-guide#access-control).
So following:
> Use access control as the leading property specifier. The only things that should come before access control are the static specifier or attributes such as @IBAction, @IBOutlet and @discardableResult.

`private` should be placed ahead of `lazy`.

Reply:
resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/17/commits/f6bd9482b6d7960d05e03b5d079bf5e2a28fe439)

Reply:
@vnntsu about the naming of branch, I think you are right. `feature/#{issue_id}` this pattern will be more readable, I will follow this for future branches.